### PR TITLE
Update README.md with example

### PR DESCRIPTION
**Pull request by kevinrizza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,29 @@ The Operator Courier is used to build, validate and push Operator Artifacts.
 It is currently in a pre-alpha stage.
 
 ## Usage
+
+### Command Line Interface
+To install the operator courier project to use from the command line, just install the latest release from [PyPI](https://pypi.org/project/operator-courier/):
+
+```bash
+$ pip3 install operator-courier
+$ operator-courier
```
Comment (aravindhp) [inline README.md]: This seems to imply executing `operator-courier` is required to install the package. I suggest leaving it out.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,29 @@ The Operator Courier is used to build, validate and push Operator Artifacts.
...
+```
+
+Once the project is installed, to generate a bundle and push it to a quay.io app registry just use `operator-courier push`
```
Comment (aravindhp) [inline README.md]: `generate a bundle` seems to imply the tool is generating the files. What is required here is a note about how the user should organize the files.

Comment (kevinrizza) [reply README.md]: @aravindhp courier does generate the bundle yaml. I can add more specifics here though to define exactly what is expected and what is generated.

Comment (aravindhp) [reply README.md]: Are we re-defining what a [bundle](https://github.com/operator-framework/operator-registry#manifest-format) is? But yes, calling out what is expected will help.

Comment (kevinrizza) [reply README.md]: @aravindhp Here we are using the word `bundle` to describe the yaml blob that is pushed up to quay (similarly to how it is used in the community-operators repo). If and when we update courier to use the nested bundles, we should update our readme and source to use the correct terms -- AFAIK there is no correct term right now for the yaml blob using the new nested format.

File: README.md (hunk 1 above)
Comment (kevinrizza) [reply README.md]: updated.

File: README.md (hunk 2 above)
Comment (kevinrizza) [reply README.md]: updated.

File: README.md [hunk 3]
```diff
@@ -18,17 +40,9 @@ We also support passing a list of strings that make up the bundle by specifying
 
 `api.build_verify_and_push($your_namespace, $your_repository, $your_release_version, $your_quay_authtoken, yamls=$your_yamls_list)`
 
-Not only that, but we also support the use of the tool as a command line interface. Just invoke `operator-courier` from a shell:
-
-```bash
-$ operator-courier verify --source_dir="./temp/bundles/couchbase/"
```
Comment (SamiSousa) [inline README.md]: Do we want to omit mention of the verify command? I agree that the focus should be on the push command, but this entirely removes mention of it.

Comment (aravindhp) [reply README.md]: That's a good point. It makes sens to call out how to verify it too.

Comment (kevinrizza) [reply README.md]: included in latest push.

**Pull request merged**